src/utils/institutions-helpers.ts

```diff
@@ -1,13 +1,13 @@
 import { CristinApiPath } from '../api/apiPaths';
-import { ConfirmedAffiliation, Contributor } from '../types/contributor.types';
+import { ConfirmedAffiliation, Contributor, PreviewContributor } from '../types/contributor.types';
 import { SimpleCustomerInstitution } from '../types/customerInstitution.types';
 import { Organization } from '../types/organization.types';
 import { API_URL } from './constants';
 import { getLanguageString } from './translation-helpers';
 
 // Find distinct unit URIs for a set of contributors' affiliations
 const unitIdToIgnore = `${API_URL}${CristinApiPath.Organization.substring(1)}/0.0.0.0`;
-export const getDistinctContributorUnits = (contributors: Contributor[]) => {
+export const getDistinctContributorUnits = (contributors: PreviewContributor[] | Contributor[]) => {
```

Comment:
Kan se ut som denne aldri brukes i noen søkekontekst, og dermed alltid burde ha contributors på den eksisterende typen? 🤔 

```suggestion
export const getDistinctContributorUnits = (contributors: Contributor[]) => {
```

Reply:
Fortsatt begge ser jeg, men ser det er pga en annen funksjon som ligger over her. Har ingen betydning her, så tenker vi lar det være sånn